module/documents/actor.mjs

```diff
@@ -85,6 +85,17 @@ export default class DhpActor extends Actor {
         this.updateSource({ prototypeToken });
     }
 
+    /**@inheritdoc */
+    async _preDelete() {
```

Comment:
I think we can do without this now (there's already a re-render in on-delete, and pruned stuff wont' show between re-renders), but if you want to keep it for data sanity, it might be good to move it to _onUpdate() and check the primary updater.